app/views/layouts/application.html.erb

```diff
@@ -14,7 +14,7 @@
     <%= stylesheet_link_tag "/chameleon/css/app.css" %>
     <%= stylesheet_link_tag "app" %>
     <%= javascript_include_tag "/chameleon/js/app.js" %>
-    <link href="//static.opensuse.org/themes/bento/images/favicon.png" rel="shortcut icon" />
+    <link href="https://software.opensuse.org/favicon.ico" rel="shortcut icon" />
```

Comment:
Anyone closer to openSUSE has a comment on these static file serving changes?

Reply:
well, the idea with the old branding was to get as much as possible from static - so that the browser has it cached when you change sites. But nowadays every app has its own theme anyway :(

Reply:
From what I can tell chameleon is not yet on static, I just know there are plans for it ;)